core/controlplane/config/templates/cloud-config-worker

```diff
@@ -845,3 +860,414 @@ write_files:
+        KUBELET_OPTS="--node-labels='alpha.kubernetes.io/nvidia-gpu-name=$NVIDIA_GPU_NAME' $KUBELET_OPTS"
+        KUBELET_OPTS="--node-labels='kube-aws.coreos.com/gpu=nvidia' $KUBELET_OPTS"
+        KUBELET_OPTS="--node-labels='kube-aws.coreos.com/nvidia-gpu-version={{.Gpu.Nvidia.Version}}' $KUBELET_OPTS"
+        echo "KUBELET_OPTS=\"$KUBELET_OPTS\"" > /etc/default/kubelet
+      fi
+
+  - path: /opt/nvidia-build/nvidia-install.sh
+    owner: root:root
+    permissions: 0755
+    content: |
+      #!/bin/bash
+      # This script is borrowed from https://github.com/Clarifai/coreos-nvidia/pull/4
+
+      if [[ $(uname -r) != *"-coreos"* ]]; then
+          echo "OS is not CoreOS"
+          exit 1
+      fi
+
+      # If we are on CoreOS by default use the current CoreOS version
+      if [[ -f /etc/lsb-release && -f /etc/coreos/update.conf ]]; then
+          source /etc/lsb-release
+          source /etc/coreos/update.conf
+
+          COREOS_TRACK_DEFAULT=$GROUP
+          COREOS_VERSION_DEFAULT=$DISTRIB_RELEASE
+          if [[ $DISTRIB_ID != *"CoreOS"* ]]; then
+              echo "Distribution is not CoreOS"
+              exit 1
+          fi
+      fi
+
+      DRIVER_VERSION=${1:-{{.Gpu.Nvidia.Version}}}
+      COREOS_TRACK=${2:-$COREOS_TRACK_DEFAULT}
+      COREOS_VERSION=${3:-$COREOS_VERSION_DEFAULT}
+
+      # this is where the modules go
+      release=$(uname -r)
+
+      mkdir -p /opt/nvidia/$DRIVER_VERSION/lib64 2>/dev/null
+      mkdir -p /opt/nvidia/$DRIVER_VERSION/bin 2>/dev/null
+      ln -sfT lib64 /opt/nvidia/$DRIVER_VERSION/lib 2>/dev/null
+      mkdir -p /opt/nvidia/$DRIVER_VERSION/lib64/modules/$release/video/
+
+      tar xvf libraries-$DRIVER_VERSION.tar.bz2 -C /opt/nvidia/$DRIVER_VERSION/lib64/
+      tar xvf modules-$COREOS_VERSION-$DRIVER_VERSION.tar.bz2 -C /opt/nvidia/$DRIVER_VERSION/lib64/modules/$release/video/
+      tar xvf tools-$DRIVER_VERSION.tar.bz2 -C /opt/nvidia/$DRIVER_VERSION/bin/
+
+      install -m 755 create-uvm-dev-node.sh /opt/nvidia/$DRIVER_VERSION/bin/
+      install -m 755 nvidia-start.sh /opt/nvidia/$DRIVER_VERSION/bin/
+      install -m 755 nvidia-insmod.sh /opt/nvidia/$DRIVER_VERSION/bin/
+      install -m 755 set-gpu-name-to-kubelet-opts.sh /opt/nvidia/$DRIVER_VERSION/bin/
+      ln -sfT $DRIVER_VERSION /opt/nvidia/current 2>/dev/null
+
+      cp -f 71-nvidia.rules /etc/udev/rules.d/
+      udevadm control --reload-rules
+
+      mkdir -p /etc/ld.so.conf.d/ 2>/dev/null
+      echo "/opt/nvidia/current/lib64" > /etc/ld.so.conf.d/nvidia.conf
+      ldconfig
+
+      echo "Configuring nvidia persistence user"
+      id -u nvidia-persistenced >/dev/null 2>&1 || \
+      useradd --system --home '/' --shell '/sbin/nologin' -c 'NVIDIA Persistence Daemon' nvidia-persistenced
+
+      cp *.service /etc/systemd/system/
+      systemctl daemon-reload
+      systemctl enable nvidia-start.service
+      systemctl start nvidia-start.service
```

Comment:
Thanks! So, basically we're copying systemd unit files written via `write_files` into `/etc/systemd/system`?
Then, it is possible to just write those units directly into the `units` section rather than copy via `write_files`, right?